cloud/scope/managedcontrolplane.go

```diff
@@ -132,6 +132,11 @@ func (s *ManagedControlPlaneScope) AdditionalTags() infrav1.Tags {
 	return tags
 }
 
+// FailureDomains returns the failure domains.
+func (s *ManagedControlPlaneScope) FailureDomains() clusterv1.FailureDomains {
```

Comment:
if you end up getting rid of `AvailabilitySetEnabled` in ClusterDescriber, make sure to allow remove it from this file (line 122)

Reply:
Done, Thanks.